### PR TITLE
Feature/complete minisankey work

### PR DESCRIPTION
**Pull request by tiagojsag**
https://github.com/Vizzuality/trase/issues/103

### CODE REVIEW

**Comment by tiagojsag** [comment]
Good to review, will fix tests tomorrow

**Comment by tiagojsag** [comment]
Seems like we have an issue on some cases, where the paths are not rendered correctly

http://localhost:8081/profile-place?nodeId=653&year=2015

image

File: frontend/scripts/pages/profile-place.page.jsx [hunk 1]
```diff
@@ -160,156 +160,47 @@ const _build = (data, { year, showMiniSankey }, store) => {
...
-    };
-
-    render(
-      <MiniSankey
-        data={tradersSankeyData}
-        targetLink="actor"
-      />,
-      document.getElementById('js-traders-sankey')
-    );
+  document.querySelectorAll('.mini-sankey-container')
```
Comment (sorodrigo) [inline frontend/scripts/pages/profile-place.page.jsx]: does this work? 👀

Comment (tiagojsag) [reply frontend/scripts/pages/profile-place.page.jsx]: seems to, but it's actually not needed, so I'll remove it

Comment (sorodrigo) [reply frontend/scripts/pages/profile-place.page.jsx]: ok I'm impressed I didn't know this. However, for the future => https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach

It's not supported on IE11 :(

**Comment by tiagojsag** [comment]
ready for re-review team

**Pull request merged**